modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/supplemental_claims/evidence_submissions_controller.rb

```diff
@@ -61,7 +61,7 @@ def supplemental_claim_uuid?
 
         def uuid_missing_error
           error = { title: 'bad_request',
-                    detail: I18n.t('appeals_api.errors.missing_uuid', appeal_type: 'SupplementalClaim') }
+                    detail: I18n.t('appeals_api.errors.missing_uuid', appeal_type: 'SC') }
```

Comment:
Since this is an error going out to our consumers, let's spell this out directly as "Supplemental Claim" instead of "SC"

Reply:
Ok, cool. Heads up though, this same logic is used for NOD and uses `appeal_type: 'NOD'`. Should we somehow make a note to change that for NOD in the future?